### DISCUSSION TITLE
none-http links are removed

### DISCUSSION THREAD

**Discussion by lancemao** [remarkjs/remark#1345] [Q&A] [answered]
in my markdown, I have some none-http links, e.g.:

```
[link](custom_scheme://mysite.com)
```

react-markdown removes my link in the output html

I also tried:

```
<a href="custom_scheme://mysite.com">link</a>
```

but it seems that react markdown doens't support pure html elements

either way works perfectly fine in vscode markdown editor

How can I solve this? Thanks!

**Comment by wooorm** [comment, marked as answer]
Hi. Please read the readme. They are unsafe. You are looking for the `urlTransform` option. Or the Security section. The readme also explains when and why “pure HTML elements” are supported, notably in Appendix A: HTML in markdown. https://github.com/remarkjs/react-markdown

**Reply by lancemao** [reply]
thanks it works great!

```jsx
<ReactMarkdown transformLinkUri={url => url}>
```

will keep the original url
